mobile/src/libs/components/button/button.tsx

```diff
@@ -2,18 +2,43 @@ import React from 'react';
 import { Pressable } from 'react-native';
 
 import { Text } from '#libs/components/components';
+import { useState } from '#libs/hooks/hooks';
 
 import { styles } from './styles';
 
 type Properties = {
   label: string;
   onPress: () => void;
+  isDisabled?: boolean;
 };
 
-const Button: React.FC<Properties> = ({ label, onPress }) => {
+const Button: React.FC<Properties> = ({
+  label,
+  onPress,
+  isDisabled = false,
+}) => {
+  const [isPressed, setIsPressed] = useState<boolean>(false);
+
+  const handleOnPress = (): void => {
+    if (isDisabled) {
+      return;
+    }
```

Comment:
pressable has disabled parameter
```suggestion
```

Reply:
we pass this boolean value as props because it can be dynamically changed by user's actions, but ive added this isDisabled boolean value to disabled parameter of pressable

Reply:
ive deleted these lines, because they are unuseful when there is disabled parameter, thank you for notification